rbc/tests/test_omnisci.py

```diff
@@ -72,9 +73,7 @@ def test_direct_call(omnisci):
     def farenheight2celcius(f):
```

Comment:
It's a general typo we should fix at some point: _fahrenheit_ is the correct orthography.

Reply:
Fixed.